squarelet/core/models.py

```diff
@@ -14,3 +15,35 @@ class Interval(Func):
 
     def __init__(self, expression, **extra):
         super().__init__(Value(expression), **extra)
+
+# Airtable Models
+from pyairtable.orm import Model, fields as F
```

Comment:
@mitchelljkotler the `pyairtable` library provides a nice ORM that handles the loading and relationship fetching for us.